wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -802,7 +804,7 @@ impl crate::Adapter<super::Api> for super::Adapter {
             .instance
             .raw
             .get_physical_device_format_properties(self.raw, vk_format);
-        let features = properties.linear_tiling_features;
+        let features = properties.optimal_tiling_features;
```

Comment:
oh wow, nice catch!

Reply:
haha, a natural consequence of testing - on Vulkan, you can pretty much do naught with linear-tiled textures on most formats. You're lucky if sampling is supported.